src/Aspire.Dashboard/Components/Layout/AspirePageContentLayout.razor.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Components.Resize;
+using Aspire.Dashboard.Resources;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components;
+
+namespace Aspire.Dashboard.Components.Layout;
+
+public partial class AspirePageContentLayout : ComponentBase
+{
+    [CascadingParameter]
+    public required ViewportInformation ViewportInformation { get; init; }
+
+    [Parameter] public required RenderFragment PageTitleSection { get; set; }
```

Comment:
Repo convention is attribute on line above

Reply:
changed